apps/src/templates/projects/ProjectAppTypeArea.jsx

```diff
@@ -37,7 +37,15 @@ const ProjectAppTypeArea = React.createClass({
     projectList: PropTypes.arrayOf(projectPropType).isRequired,
     numProjectsToShow: PropTypes.number.isRequired,
     galleryType: PropTypes.oneOf(['personal', 'class', 'public']).isRequired,
-    navigateFunction: PropTypes.func.isRequired
+    navigateFunction: PropTypes.func.isRequired,
+    isDetailView: PropTypes.bool.isRequired,
```

Comment:
this could use a comment explaining that this means we are looking at only one project type